### PR TITLE
Feat:History watchlater page

### PR DESCRIPTION
**Pull request by ramalingaa**
Added History and Watch later page

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
Your UI is functional
Suggestions
- some of the thumbnail images are missing
- add loader for deleting videos from history and watch later pages

PR is ready to merge

File: src/App.css [hunk 1]
```diff
@@ -634,4 +635,37 @@ a {
...
+  .clear-history {
+    position: absolute;
+    top: -2rem;
+    left: 4.5%;
+    border: 0;
+    background-color:transparent;
+    
+    text-decoration: underline;
+    padding: 0.5rem;
+  } */
```
Comment (romabulani) [inline src/App.css]: You can remove the comments if not required in future

File: src/frontend/components/authentication/Login.jsx [hunk 2]
```diff
@@ -20,8 +20,9 @@ const Login = () => {
             
                 const response = await axios.post("/api/auth/login",userData)
                 if(response.status === 200){
-                    localStorage.setItem("JWT_TOKEN_MOZI",response.data.encodedToken)
-                    localStorage.setItem("USER_PROFILE_MOZI",JSON.stringify(response.data.foundUser))
+                    localStorage.setItem("mozi",JSON.stringify({"JWT_TOKEN_MOZI":response.data.encodedToken, "USER_PROFILE_MOZI":response.data.foundUser.firstName}))
+                    // localStorage.setItem("JWT_TOKEN_MOZI",response.data.encodedToken)
+                    // localStorage.setItem("USER_PROFILE_MOZI",JSON.stringify(response.data.foundUser))
```
Comment (romabulani) [inline src/frontend/components/authentication/Login.jsx]: Here also, if not required, remove the comments

**Review by romabulani — APPROVED** [review]
Code is clean and well written, good to merge
Suggestions:
- Remove comments if not required in future
- Thumbnails of some videos are missing in UI

File: src/frontend/components/authentication/Login.jsx (hunk 2 above)
Comment (ramalingaa) [reply src/frontend/components/authentication/Login.jsx]: Commented code is removed

**Pull request merged**